app/Http/Controllers/PayloadController.php

```diff
@@ -143,16 +149,25 @@ public function destroy(Webhook $webhook, Payload $payload)
         $this->authorize('delete', [$payload, $webhook]);
         if ($payload->conditions->count() > 0) {
             return redirect()->back()
-                ->with('messageFail', 'This payload has some conditions to be related with, please delete them first');
+                ->with('messageFail', [
+                    'status' => 'Delete failed',
+                    'message' => 'This payload has some conditions to be related with, please delete them first'
```

Comment:
**Reporter**: CHECKSTYLE</br>**Rule**: Framgia.Arrays.ArrayDeclaration.NoComma</br>**Severity**: error</br>**File**: app/Http/Controllers/PayloadController.php</br>**Line**: 154</br>Each line in an array declaration must end in a comma